scripts/calendar/hindu_generator.py

```
@@ -0,0 +1,783 @@
+    "Bhadrapad",  # (120°-150°) - Leo (4)
+    "Ashwin",  # (150°-180°) - Virgo (5)
+    "Kartik",  # (180°-210°) - Libra (6)
+    "Margashirsh",  # (210°-240°) - Scorpio (7)
+    "Paush",  # (240°-270°) - Sagittarius (8)
+    "Magh",  # (270°-300°) - Capricorn (9)
+    "Phalgun",  # (300°-330°) - Aquarius (10)
+    "Chaitra",  # (330°-360°) - Pisces (11)
+]
+
+
+def _lunar_month(jd):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_lunar_month`

(ANN202)